src/igvfd/schemas/donor.json

```diff
@@ -67,6 +67,20 @@
             },
             "maxItems": 2
         },
+        "traits": {
+            "title": "Phenotypic traits",
+            "description": "A list of known phenotypic traits of the donor",
+            "type": "array",
```

Comment:
missed that, but @hitz pointed out that we should probably not define traits in three places, instead define it in donor only, and you can update the description in children's schemas (like human and rodent donors)